app/ui/legacy/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.kt

```diff
@@ -130,6 +130,21 @@ class RecipientPresenter(
             return true
         }
 
+        if (!recipientMvpView.recipientToIsValid()) {
```

Comment:
The order of the checks should be rearranged as follows:

Place the following check after the condition that verifies if `toAddresses`, `ccAddresses`, and `bccAddresses` are empty:

```kotlin
if (toAddresses.isEmpty() && ccAddresses.isEmpty() && bccAddresses.isEmpty()) {
    recipientMvpView.showNoRecipientsError()
    return true
}
```

Doing so ensures that the user is informed to add at least one recipient, rather than receiving a misleading message about invalid input in the fields.